### ISSUE TITLE
Code folding

### ISSUE THREAD

**Issue by Phalangers** [sashaweiss/vscode_block_travel#1]
The jumps don't take into account code folding.
It opens folded code.

Reproduce:
Ctrl-k Ctrl-0 : fold all
Jump around

**Comment by sashaweiss** [comment]
Hey, so sorry I never replied. I must have missed the email!

I'm not actively developing this project anymore, but if I get a chance I'll come back to this issue! Thanks for the note. I hope you were able to find a workaround in the meantime!

**Comment by sashaweiss** [comment]
Closing for now - if I get a chance to come back I'll reopen.

**Comment by monk-time** [comment]
Stumbled upon the same issue. This bug forces me either to stop using the folding feature or this extension, and that's very unfortunate.

Don't know why this is closed, I guess some people have quite a peculiar workflow for managing their trackers. :)

**Comment by sashaweiss** [comment]
Fair point - happy to leave it reopened, but unfortunately I probably won't have time to fix it anytime soon. PRs welcome!
